Echo.json5

```diff
@@ -107,7 +107,7 @@
       name: 'StripeStagingPublishableKey',
       content: '${StripeStagingPublishableKey}',
     },
-    { name: 'KillSwitchBuildMinimum', content: '5.0.0' },
+    { name: 'KillSwitchBuildMinimum', content: '6.5.0' },
```

Comment:
Not so bad -- so the flow is: pull down this JSON on app launch, check for the `KillSwitchBuildMinimum` and compare against app version and then prompt user to upgrade.